test/ViewsTest.cpp

```diff
@@ -265,3 +265,91 @@ TEST(Views, verifyLineByLineWorksWithChunksBiggerThanLines) {
   ++iterator;
   ASSERT_EQ(iterator, lineByLineGenerator.end());
 }
+
+// _____________________________________________________________________________
+TEST(Views, RvalueView) {
+  // A simple struct that knows if it has been moved from.
+  struct MoveTracker {
+    bool wasMoved_ = false;
+    MoveTracker() = default;
+    MoveTracker(const MoveTracker&) = default;
+    MoveTracker& operator=(const MoveTracker&) = default;
+    MoveTracker(MoveTracker&& rhs)
+        : wasMoved_{std::exchange(rhs.wasMoved_, true)} {}
+    MoveTracker operator=(MoveTracker&& rhs) {
+      wasMoved_ = std::exchange(rhs.wasMoved_, true);
+      return *this;
+    }
+  };
+
+  std::vector<MoveTracker> vec;
+  vec.resize(13);
+
+  std::vector<MoveTracker> target;
+  {
+    // Move the first 5 elements via an `RvalueView` + ql::ranges::copy.
+    auto view = ad_utility::RvalueView{vec};
```

Comment:
Sonarcloud will probably complain about this, you can just write the type instead of auto here. Same for the occurrences below.